src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -660,6 +659,18 @@ export default class PoseEditMode extends GameMode {
 
         this._folder = initialFolder || FolderManager.instance.getFolder(this._puzzle.folderName);
 
+        // now that we have made the folder check, we can set _targetPairs. Used to do this
+        // above but because NuPACK can handle pseudoknots, we shouldn't
+        for (let ii = 0; ii < targetSecstructs.length; ii++) {
+            if (this._folder.name === 'NuPACK'
```

Comment:
At the very least, use `NuPACK.NAME` rather than the static string. Even better though would be adding a `canPseudoknot` to `Folder` like `canScoreStructures`, `canFoldWithBindingSite`, etc. and require puzzles with pseudoknotting to use NuPACK via `Puzzle.canUseFolder`. Or another option might be going one step further and when you actually go to do the folding, to only use pseudoknots if the flag is on AND the engine supports it - this is somewhat of a policy decision though. Or maybe change can_pseudoknot to have three states (don't do it, do it if the current engine supports it, and only use engines that support it)

Same deal the other locations where this is referenced